wfpr_modules/github.com/icgc-argo-workflows/argo-qc-tools/cutadapt@0.1.1/README.md

```diff
@@ -0,0 +1,97 @@
+# Cutadapt Quality Control workflow
+
+This tool runs Cutadapt for the sole purpose of extracting information about
+adapter contamination. It does not output the trimmed FASTQs by default.
+
+## Prerequisites
+
+* Nextflow, tested on 21.04.1
+* Docker, tested on V 3.5.1
+
+## Supported platforms
+
+Because this runs in a Docker container, any Linux or Linux-like environment is
+sufficient.
+
+## Running
+
+With parameters on the command line:
+```
+nextflow run main.nf --input_R1 tests/input/TCRBOA7-T-RNA.250reads.read1.fastq.gz --input_R2 tests/input/TCRBOA7-T-RNA.250reads.read2.fastq.gz --publish_dir testmyoutput
+```
+
+With a parameter file:
+
+```
+nextflow run main.nf -params-file params.json
+```
+
+## Input parameters
+
+**Required**:
+
+* `input_R1`: Read1 fastq file. This is assumed to be in a gzipped form.
+* `input_R2`: Read2 fastq file. This is assumed to be in a gzipped form.
+* `publish_dir`: the final location for the results.
+
+**Optional**:
+
+* `read1_adapter` : Override the adapter for read 1. Default: "AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC"
+* `read2_adapter` : Override the adapter for read 2. Default: "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT"
```

Comment:
This is the old version of the workflow. Please delete.

Reply:
Good catch. already removed it.